reactivesocket-cpp/src/ReactiveStreamsCompat.h

```diff
@@ -2,7 +2,7 @@
 
 #pragma once
 
-#include "reactive-streams-cpp/ReactiveStreams.h"
+#include <ReactiveStreams.h>
```

Comment:
It wouldn't change the class names. It would align with the project name so I don't see it as a coding convention change.